modules/gitops-additional-permissions-for-cluster-config.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assembly:
+//
+// * configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-additional-permissions-for-cluster-config_{context}"]
+= Providing additional permissions for cluster configuration
+
+You can add permissions to an Argo CD instance to manage cluster configuration. 
+
+.Procedure
+
+Create a ClusterRole with additional permissions and then create a new ClusterRoleBinding to associate the ClusterRole with a ServiceAccount:
```

Comment:
```suggestion
You can add permissions to an Argo CD instance to manage cluster configuration. Create a cluster role with additional permissions and then create a new cluster role binding to associate the cluster role with a service account.

.Procedure

```

Reply:
The SSG ask us to avoid these types of lead-in sentences: https://redhat-documentation.github.io/supplementary-style-guide/#lead-in-sentences